modules/core/id_tag/src/Plugin/Field/FieldWidget/IdTagWidget.php

```diff
@@ -38,7 +38,7 @@ public function formElement(FieldItemListInterface $items, $delta, array $elemen
     $tag_type = $items[$delta]->type ?? NULL;
 
     // Get the current asset bundle.
-    $bundle = $items->getParent()->getEntity()->bundle();
+    $bundle = $items->getEntity()->bundle();
```

Comment:
Note to self: need to do some manual testing to ensure this change doesn't break anything. _I think_ this will work, but the types were very dynamic and complex here. If this breaks something we can revert and add this error to our baseline.